source/documentation/appendix-docs/data-movement-form.md

```diff
@@ -0,0 +1,269 @@
+to access the template, please contact the
+[Analytical Platform team](mailto:analytical_platform@digital.justice.gov.uk).
+
+#### Technical migration form
+
+Please send your form as an attachment to the
+[Analytical Platform team](mailto:analytical_platform@digital.justice.gov.uk).
+
+#### Is this a one-off or recurring data movement?
+
+#### When is the data movement expected to occur?
+
+#### What is the intended frequency of the data movement?
+
+#### When is the first data movement expected to occur?
+
+#### Will the data movement be in place for a fixed period of time?
+
+#### When will the final data movement occur?
+
+#### When will the data movement request be reviewed?
+
+Recurring data movements that occur indefinitely should be reviewed at least
+every two years.
+
+## Data retention
+
+#### How long will the data be retained on the Analytical Platform?
+
+#### Is this in line with departmental and Analytical Platform data retention policies?
+
+See the [data retention](../../information-governance.html#data-retention)
+section.
+
+## Information Asset Owner (IAO)
+
+All data in MoJ is assigned an Information Asset Owner (IAO). The IAO is
+responsible for the registration, labelling, storage, transfer and retention of
+that data.
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+    You must obtain approval from the relevant IAO for all data
+movements.
+  </strong>
+</div>
+
+#### Does the data movement require approval by an Information Asset Owner (IAO)?
+
+#### IAO name
+
+#### IAO email address
+
+#### Has the IAO approved this data movement?
+
+## Senior Information Risk Officer (SIRO)
+
+Information on the responsibilities of SIROs can be found on the
+[intranet](https://intranet.justice.gov.uk/guidance/knowledge-information/protecting-information/information-assurance-roles/).
+
+SIRO approval may be required depending on local information assurance policies.
+SIRO approval is usually required when a data movement involves non-routine
+risks. The relevant IAO should be able to advise if SIRO approval is also
+required.
+
+#### Does the data movement require approval by a Senior Information Risk Officer (SIRO)?
+
+#### SIRO name
+
+#### SIRO email address
+
+#### Has the SIRO approved this data movement?
+
+## Information assurance
+
+#### Have you discussed the data movement with the relevant information security team(s)?
+
+Some information security teams that may be relevant are:
+
+- [MoJ HQ cybersecurity team](mailto:cybersecurity@digital.justice.gov.uk)
+- [HMCTS information assurance team](https://intranet.justice.gov.uk/about-hmcts/digital-change/information-assurance-and-data-security/)
+- [HMPPS information management and security team](mailto:informationmgmtsecurity@noms.gsi.gov.uk)
```

Comment:
"Have you discussed" seems like a leading question, and puts pressure on the person writing this out. I think this guidance can help by saying when you should seek guidance before submitting this form, and when to leave it to the discretion of the AP team.

Reply:
I think it was meant to be leading but I get the point that it may not be necessary in all cases.

Reply:
I've added some pointers in 56f5546. I'm not sure I'm know enough myself about when you do and don't need to contact these teams to articulate it more clearly.